framework/source/class/qx/ui/layout/Grid.js

```diff
@@ -995,7 +1018,11 @@ qx.Class.define("qx.ui.layout.Grid",
         // increment the preferred column sizes.
         if (prefSpanWidth < hint.width)
         {
-          if (!qx.lang.Object.isEmpty(colFlexes)) {
+          // Do not adapt column widths to the width
+          // of the spanning cell if allowGrowSpannedCellWidth property
+          // is set to false
+          // See https://github.com/qooxdoo/qooxdoo/issues/9871
+          if (this.getAllowGrowSpannedCellWidth() === false || !qx.lang.Object.isEmpty(colFlexes)) {
```

Comment:
What's here isn't wrong, but `if (! this.getAllowGrowSpannedCellWidth() || ...` vs `=== false` is more common

Reply:
done.